.pre-commit-config.yaml

```diff
@@ -25,6 +25,7 @@ repos:
     -   id: trailing-whitespace
         exclude: github_scripts/pelican_protocol.patch
     -   id: end-of-file-fixer
+        exclude: ".*/parameters.json"
```

Comment:
Instead of doing this, I'd prefer we add the new line at the end of the generated `parameter.json`